src/Features/Core/Portable/CodeFixes/CodeFixService.cs

```diff
@@ -392,6 +398,9 @@ private bool TryGetWorkspaceFixersPriorityMap(Document document, [NotNullWhen(tr
                 {
                     cancellationToken.ThrowIfCancellationRequested();
 
+                    if (highPriority != null && highPriority != fixer.IsHighPriority)
+                        continue;
```

Comment:
code fixing, refactoring, and analyzers all effectively have a way to say: if you want high-pri or normal-pri, i'll limit to that set.